docs/_instructions/dsannotations-options.md

```diff
@@ -18,7 +18,7 @@ The following options are supported:
 
 |option||
 |-|-|
-|inherit|use DS annotations found in the class hierarchy of the component class|
+|inherit|use DS annotations found in the class hierarchy of the component class. *WARNING:* Annotations are not inherited form the component's super classes by default. The problem is that bnd cannot in all cases see super classes (and interfaces) because the super class does not have to be available in the build. In runtime you are sure the super class is there, at build time you do not have that luxury. So to make sure, always define your annotations on the actual component class. |
```

Comment:
Same grammar fix needed here: not inherited _from_

Reply:
and if the super classes are from imported packages, then the actual super classes at runtime can differ from the super classes seen at build time.

Reply:
That's a much bigger problem than "not available at build time", which is in my experience not likely with a correctly set up build.